adb_graphics/default_specs.yml

```diff
@@ -740,6 +740,56 @@ icmr: # Ice Water Mixing Ratio
         - ICMR_P0_L105_{grid}
         - CIMIXR_P0_L105_{grid}
       prs: ICMR_P0_L100_{grid}
+icprb: # Icing Probability
+  500ft: &icprb
+    clevs: !!python/object/apply:numpy.arange [5, 86, 10]
+    cmap: gist_ncar
+    colors: icprb_colors
+    ncl_name: ICPRB_P0_L102_{grid}
+    ticks: 10
+    transform: conversions.percent
+    unit: "%"
+    vertical_index: 0   # grib2 vertical levels are in meters,
+                        # so using index here to avoid errors from conversions
+  1000ft:
+    <<: *icprb
+    vertical_index: 1
+  9000ft:
+    <<: *icprb
+    vertical_index: 17
+  15000ft:
+    <<: *icprb
+    vertical_index: 29
+  max:
+    <<: *icprb
+    transform: 
+      funcs: field_column_max
+      kwargs:
+        variable: icprb
+icsev: # Icing Severity
+  500ft: &icsev
+    clevs: [0.0, 0.5, 1.0, 2.0, 3.0]
```

Comment:
It might be good to add a quick note here that mentions the numerical values get mapped to categorical labels in maps.py and that the original data is delivered out of order, so the 0.5 value is handled in maps.py, too.